Gemfile

```diff
@@ -56,6 +56,7 @@ gem "inifile",                        "~>3.0",         :require => false
 gem "jbuilder",                       "~>2.5.0" # For the REST API
 gem "manageiq-api-client",            "~>0.1.0",       :require => false
 gem "mime-types",                     "~>2.6.1",       :require => "mime/types/columnar"
+gem "money-rails",                    "~>1"
```

Comment:
It would be nice if we could find a way to delay load this and the initializer until someone actually makes a chargeback call.  cc @jrafanie